eclair-core/src/test/scala/fr/acinq/eclair/StartupSpec.scala

```diff
@@ -273,4 +273,26 @@ class StartupSpec extends AnyFunSuite {
     assert(nodeParamsAttempt2.isSuccess)
   }
 
+  test("NodeParams should fail when server.public-ips addresses or server.port are invalid") {
```

Comment:
nit: there are whitespace formatting issues, can you run `Ctrl + Alt + L`?

Reply:
fixed in [ff2ac04](https://github.com/ACINQ/eclair/pull/2234/commits/ff2ac0419ff96c9dabc00e68add94c35e0ad8344)